README.md

```diff
@@ -349,7 +349,7 @@ like the following:
     {
       "name": "gamma",
       "type": "boolean",
-      "description": "True or false?"
+      "description": "true or false?"
```

Comment:
Can't merge this: this definition is the result of parsing the above function, where "True or false?" is capitalized. Resubmit with only syntax highlighting changes and I can accept.